### PR TITLE
OLS-307 Expose selected model and provider through Prometheus metrics

### PR DESCRIPTION
**Pull request by tisnik**
## Description

Expose selected model and provider through Prometheus metrics

## Type of change

- [ ] Refactor
- [x] New feature
- [ ] Bug fix
- [ ] CVE fix
- [ ] Optimization
- [ ] Documentation Update
- [ ] Configuration Update
- [ ] Bump-up dependent library

## Related Tickets & Documents

- Related Issue #[OLS-307](https://issues.redhat.com//browse/OLS-307)

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [x] PR has passed all pre-merge test jobs.
- [ ] If it is a core feature, I have added thorough tests.

### CODE REVIEW

File: ols/app/metrics/metrics.py [hunk 1]
```diff
@@ -23,5 +23,12 @@
     "llm_token_received_total", "LLM tokens received", ["provider", "model"]
 )
 
+# expose selected provider and model
+# (these are represented by counters, but the only meaning is presence of label)
+selected_provider_total = Counter(
+    "selected_provider", "Selected provider", ["provider"]
+)
+selected_model_total = Counter("selected_model", "Selected model", ["model"])
```
[the remark this answers is not in the document]
Comment (tisnik) [reply ols/app/metrics/metrics.py]: updated

File: ols/app/metrics/metrics.py [hunk 2]
```diff
@@ -23,5 +23,10 @@
     "llm_token_received_total", "LLM tokens received", ["provider", "model"]
 )
 
+# expose selected provider and model
+# (these are represented by counters, but the only meaning is presence of label)
```
Comment (raptorsun) [inline ols/app/metrics/metrics.py:27]: a small detail, the info is a "gauge" type set always to 1.0.

**Review by raptorsun — APPROVED** [review]

**Pull request merged**